packages/react-query/tsup.config.ts

```diff
@@ -0,0 +1,4 @@
+import { defineConfig } from 'tsup'
+import { options } from '@suspensive/tsup'
+
+export default defineConfig(options)
```

Comment:
currently we use like this, but I think customize is needed after. but not now

Reply:
I agree

Tsup is still working well without any customization it is really cool